lib/kitchen/driver/dokken.rb

```diff
@@ -604,6 +607,9 @@ def wait_running_state(name, v)
       end
 
       def chef_container_name
+        if config[:platform] != ""
```

Comment:
what do you think about using a ternary here?

Reply:
Agreed. I have pushed a new commit.